isso/ext/notifications.py

```diff
@@ -76,7 +76,14 @@ def __init__(self, isso):
 
         self.isso = isso
         self.conf = isso.conf.section("smtp")
-        self.public_endpoint = isso.conf.get("server", "public-endpoint") or local("host")
+        self.public_endpoint = isso.conf.get("server", "public-endpoint")
+        # rstrips potential trailing '/', without having to `repr` the `local` object.
+        if self.public_endpoint:
+            if self.public_endpoint.endswith('/'):
```

Comment:
Ask for forgiveness, not permission: just try to rstrip('/') rather than checking if there is a trailing slash first.